3p/recaptcha.js

```diff
@@ -80,6 +86,13 @@ window.initRecaptcha = function() {
     return;
   }
 
+  // Get our mode from the iframe name attribture
+  dev().assert(
+      dataObject.mode,
```

Comment:
is mode only used for testing purpose? if so, why do we need to force it?

Reply:
Done.